infrastructure/main.tf

```diff
@@ -24,6 +24,7 @@ locals {
 
 module "consumer" {
   source   = "git@github.com/hmcts/terraform-module-webapp.git"
+  source   = "git@github.com:contino/moj-module-webapp.git"
```

Comment:
The HMCTS version is empty

Reply:
oh so terraform applies the first non empty one?

Reply:
No, this is a mistake. HMCTS version should have been removed.